tutorials/util.py

```diff
@@ -0,0 +1,43 @@
+"""Common util functions used in weldx tutorials."""
+from pathlib import Path
+
+tutorials_dir = Path(__file__).parent.absolute()
+
+
+def download_tutorial_input_file():
+    from urllib.request import urlretrieve
+
+    url = (
+        "https://github.com/BAMWelDX/IIW2021_AA_CXII/blob/weldx_0.5.0"
+        "/single_pass_weld.weldx?raw=true "
+    )
+    sha256sum = "29e4f11ef1185f818b4611860842ef52d386ad2866a2680257950f160e1e098a"
```

Comment:
can we reuse this in `conf.py`? The code looks identical?

Reply:
Think so, this was part of the PR from @marscher  IIRC

Reply:
It is identical. Once we have mode the tutorials to the docs dir, we do not need this dupe any more.